local/build.go

```diff
@@ -103,8 +103,11 @@ func (s *composeService) build(ctx context.Context, project *types.Project, opts
 			Driver: d,
 		},
 	}
+	ctx2, cancel := context.WithCancel(context.TODO())
```

Comment:
nit: maybe a more explicit context name or a full comment to explain why we can't share the context here

Reply:
Done